fms/fms_io.F90

```diff
@@ -137,7 +137,7 @@ module fms_io_mod
 integer, parameter, private :: max_fields=400
 integer, parameter, private :: max_axes=40
 integer, parameter, private :: max_atts=20
-integer, parameter, private :: max_domains = 10
+integer, parameter, private :: max_domains = 1000
```

Comment:
1000 is a bit extreme for the max_domain limit - is this a realistic limit?

Reply:
Would 100 be OK?

Reply:
Yes.  As the fms_io has been superseded by fms2_io and has moved into available but unsupported mode, keeping a sane limit is preferable.  A transition guide has been provided and can be found [here](https://github.com/NOAA-GFDL/FMS/blob/main/fms2_io/readme.md).  More detailed information on fms2_io interfaces can also be found on the documentation site:  http://noaa-gfdl.github.io/FMS/group__fms2__io.html

Reply:
@danholdaway - If you think 100 will cover your needs, can you make the change so we can review and get this ready for merge?

Reply:
I've reduced to 100. I don't think there are any JEDI applications that instantiate more than 100 simultaneous geometry objects.